### DISCUSSION TITLE
achievement ideas

### DISCUSSION THREAD

**Discussion by fernandounger** [community/community#21038] [Other Feature Feedback, Questions, & Ideas]
- Your first contribution to an organization

**Comment by RodrigoDornelles** [comment]
I think an achievement is interesting for those who give 1,000 stars to other people's repositories. 🤠

**Reply by Griphcode** [reply]
Sounds like a good achievement

**Comment by douglasfdev** [comment]
can have a achievement to make +50 repos publics or something like that
